malariagen_data/anoph/snp_frq.py

```diff
@@ -1252,6 +1255,398 @@ def snp_genotype_allele_counts(
 
         return df_snps.query(snp_query)
 
+    @check_types
+    @doc(
+        summary="""
+            DataFrame of SNP frequencies.
+        """,
+        parameters=dict(
+            list_of_snps="List of SNP positions.",
+        ),
+    )
+    def snp_freqs(
+        self,
+        *,
+        list_of_snps: List[int],
+        contig: base_params.contig,
```

Comment:
```suggestion
    def snp_freqs(
        self,
        *,
        list_of_snps: List[int],
        contig: base_params.contig,
        sample_sets=None,
        sample_query=None,
        site_mask=None
```

Reply:
just exposing those parameters... wanted to see what speed is like for smaller cohorts